src/components/adminDashboard/components/esLink/esLink.component.js

```diff
@@ -0,0 +1,179 @@
+                        noData: false
+                    }
+                };
+
+                var commonGridOptions = {
+                    enableColumnMenus: false,
+                    enableRowSelection: false,
+                    enableRowHeaderSelection: false,
+                    multiSelect: false,
+                    rowHeight: 35,
+                    selectionRowHeaderWidth: 35
+                };
+                var translateFilter = $filter('translate');
+
+                _initGrid();
+
+                self.selectEducatorRow = function (rowData) {
+                    if (self.gridEducatorApi.selection.selectRow) {
+                        self.gridEducatorApi.selection.selectRow(rowData);
+                        self.selectedEducator = rowData;
+                    }
+                };
+                self.selectStudentRow = function (rowData) {
+                    if (self.gridStudentApi.selection.selectRow) {
+                        self.gridStudentApi.selection.selectRow(rowData);
+                        self.selectedStudent = rowData;
+                    }
+                };
+                self.getEducatorsSearchResults = function (queryTerm) {
+                    AdminSearchService.getSearchResultsByTerm(queryTerm).then(_educatorsSearchResults);
+                };
+                self.getStudentsSearchResults = function (queryTerm) {
+                    AdminSearchService.getSearchResults(queryTerm).then(_studentsSearchResults);
+                };
+
+                self.link = function () {
+                    self.startLoader = true;
+                    if (!(self.selectedEducator && self.selectedStudent)) {
+                        $log.error("Must select student and educator");
+                        return;
+                    }
+                    var studentEducatorAppNames = _getStudentEducatorAppNames();
+                    if (!studentEducatorAppNames) {
+                        $log.error("Must provide educator and student app data");
+                        return;
+                    }
+                    var student = self.selectedStudent;
+                    var educator = self.selectedEducator;
+                    var invitationObj = ESLinkService.createInvitationFactory(educator.uid, student.uid, educator.name, student.email, educator.email, student.name,
+                        studentEducatorAppNames.educator, studentEducatorAppNames.student);
+
+                    ESLinkService.link(invitationObj).then(_linkSuccess, _linkError);
+
+                };
+                function _linkSuccess() {
+                    _endLoading();
+                    var msg = translateFilter('ADMIN.ESLINK.LINK_SUCCEEDED');
+                    _showNotification('success', msg);
+                }
+
+                function _linkError(err) {
+                    _endLoading();
+                    // var msg = "<span>translateFilter('ADMIN.ESLINK.LINK_FAILED')</span></br>+err";
```

Comment:
currently, the service for creating toasters do not support passing an HTML string, that way I kept the string in a comment.
in any case I remove it.